packages/manager/modules/log-to-customer/src/tiles/controller.js

```diff
@@ -100,10 +100,12 @@ export default class LogToCustomerTileCtrl {
   }
 
   getDataStreams() {
+    this.loadingLogAccountStreams = true;
     return this.LogToCustomerService.getDataStreams().then(
       ({ streams, streamCount }) => {
         this.logAccountStreams = streams;
         this.hasAtLeastOneDataStream = streamCount > 0;
+        this.loadingLogAccountStreams = false;
       },
     );
```

Comment:
To hide spinner in case of error
```suggestion
    this.loadingLogAccountStreams = true;
    return this.LogToCustomerService.getDataStreams().then(
    ({ streams, streamCount }) => {
    this.logAccountStreams = streams;
    this.hasAtLeastOneDataStream = streamCount > 0;
    },
    ).finally( () => {
    this.loadingLogAccountStreams = false;
    });
```